cylc/flow/platforms.py

```diff
@@ -88,7 +88,7 @@ def get_platform(
             get_platform() or platform_from_job_info(), but to the
             user these look the same.
     """
-    if task_conf is None or isinstance(task_conf, str):
+    if task_conf is None or isinstance(task_conf, str):  # noqa: SIM 114
```

Comment:
I'm not sure that the QA is right about the suggested simplification.

Reply:
It isn't, there is an issue open on flake8-simplify.